src/lib/__tests__/InstantSearch-integration-test.ts

```diff
@@ -0,0 +1,46 @@
+import { getByText, fireEvent } from '@testing-library/dom';
+import instantsearch from '../../index.es';
+import { configure } from '../../widgets';
+import { connectConfigure } from '../../connectors';
+import { createSearchClient } from '../../../test/mock/createSearchClient';
+
+describe('InstantSearch integration', () => {
+  it('renders', () => {
```

Comment:
test name should be about configure & onStateChange

Reply:
oops, it was a temporary name. will fix it

Reply:
https://github.com/algolia/instantsearch.js/pull/4555/commits/da451436ec545b5d724681014abd2bdc31e1440c